third_party/jpeg/jpeg.BUILD

```diff
@@ -5,6 +5,8 @@ licenses(["notice"])  # custom notice-style license, see LICENSE.md
 
 exports_files(["LICENSE.md"])
 
+load("@//third_party:common.bzl", "expand_header_template")
```

Comment:
It's currently not possible for other Bazel projects to depend on TensorFlow for other reasons, such as our ./configure being mandatory. So the way I see it, we can just add issue 1248 to our list of blockers. We can fix this later.